pkg/controllers/management/feature/feature_handler.go

```diff
@@ -16,13 +17,15 @@ type handler struct {
 	featuresClient managementv3.FeatureClient
 	tokensLister   managementv3.TokenCache
 	tokenEnqueue   func(string, time.Duration)
+	mgmt           *config.ManagementContext
 }
 
-func Register(ctx context.Context, wContext *wrangler.Context) {
+func Register(ctx context.Context, wContext *wrangler.Context, management *config.ManagementContext) {
 	h := handler{
 		featuresClient: wContext.Mgmt.Feature(),
 		tokensLister:   wContext.Mgmt.Token().Cache(),
 		tokenEnqueue:   wContext.Mgmt.Token().EnqueueAfter,
+		mgmt:           management,
```

Comment:
Common pattern is to have NodeDriverLister/NodeDriverClient here than passing entire management context (look above at tokensLister for example)

Reply:
@kinarashah I just updated the code. Many thanks for the input!